common-libs/common/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@affinidi/common",
-  "version": "2.0.0-beta.4",
+  "version": "2.0.0-beta.5",
```

Comment:
Please also update the references in dependent packages (sdk\core etc).

No need to update dependent packages versions, since behavior is not changed there; just update their `@affinidi/common` dependencies to beta.5. This only affects package.json; package-lock.json does not refer to @affinidi/common.

Reply:
done